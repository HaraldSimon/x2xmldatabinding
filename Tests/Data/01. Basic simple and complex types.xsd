<?xml version="1.0" encoding="UTF-8"?>
<!-- edited with XMLSpy v2007 sp1 (http://www.altova.com) by M. van Renswoude (XÂ²Software) -->
<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" attributeFormDefault="unqualified">
	<xs:element name="TestElement">
		<xs:annotation>
			<xs:documentation>Simple test element</xs:documentation>
		</xs:annotation>
		<xs:complexType>
			<xs:sequence>
				<xs:element name="SimpleType" type="xs:integer"/>
				<xs:element name="ComplexType" type="TestComplexType"/>
			</xs:sequence>
		</xs:complexType>
	</xs:element>
	<xs:complexType name="TestComplexType">
		<xs:sequence>
			<xs:element name="AnotherSimpleType" type="xs:string"/>
		</xs:sequence>
	</xs:complexType>
</xs:schema>
