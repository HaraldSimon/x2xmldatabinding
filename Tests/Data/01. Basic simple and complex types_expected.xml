<?xml version="1.0" encoding="UTF-8"?>
<DataBindingResult xsi:noNamespaceSchemaLocation="F:\Archive\2007\XMLDataBinding\Tests\XSD\DataBindingResult.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
	<Schemas>
		<Schema>
			<Name>01. Basic simple and complex types</Name>
			<Items>
				<Item>
					<Name>TestElement</Name>
					<ItemType>Interface</ItemType>
					<Interface>
						<InterfaceType>Interface</InterfaceType>
					</Interface>
				</Item>
				<Item>
					<Name>TestComplexType</Name>
					<ItemType>Interface</ItemType>
					<Interface>
						<InterfaceType>ComplexType</InterfaceType>
					</Interface>
				</Item>
			</Items>
		</Schema>
	</Schemas>
</DataBindingResult>
